models/mrsegmentator/config/default.yml

```diff
@@ -0,0 +1,34 @@
+general:
+  data_base_dir: /app/data
+  version: 1.0.0
+  description: MRSegmentator default config (dicom to dicom)
+
+execute:
+- DicomImporter
+- NiftiConverter
+- MRSegmentatorMLRunner
+- DsegConverter
+- DataOrganizer
+
+modules:
+  DicomImporter:
+    source_dir: input_data
+    import_dir: sorted_data
+    sort_data: true
+    meta: 
+      mod: '%Modality'
+
+  MRSegmentatorMLRunner:
+    use_fast_mode: false
+
+  DsegConverter:
+    model_name: MRSegmentator
+    body_part_examined: WHOLEBODY
+    source_segs: nifti:mod=seg
+    skip_empty_slices: True
+    target_dicom: dicom:mod=ct|mr
+  
+  DataOrganizer:
+   targets:
+   - dicomseg:mod=seg-->[i:sid]//MRSegmentator.seg.dcm
+   - log-->[i:sid]/logs/[d:module]/[filename]
```

Comment:
Logs are usually not exported by default.

Reply:
Thanks! I will remove the logging from the default config.